modules/module-extras.php

```diff
@@ -25,6 +25,7 @@
 	'seo-tools/jetpack-seo-posts.php',
 	'simple-payments/simple-payments.php',
 	'verification-tools/verification-tools-utils.php',
+	'calypso-redirects.php'
```

Comment:
Maybe this should be placed a bit downer, behind the check for `if ( ! Jetpack::is_active() && ! Jetpack::is_development_mode() ) {` since Jetpack needs to be connected to WordPress.com for the Calypso redirects to make sense.

Reply:
+1 I think we should also be checking for whether or not the user is linked somewhere.  For a secondary admin that is not linked to a wpcom user, these redirects will land them in a logged-out view of wordpress.com, with no clear reason why.